android/expoview/src/main/java/host/exp/exponent/notifications/PushNotificationHelper.java

```
@@ -239,14 +242,66 @@ public Intent provide() {
+                if (!isMultiple && bodyObject.has("_richContent")) {
+                  final JSONObject richContent = bodyObject.getJSONObject("_richContent");
+                  if (richContent.has("image")) {
+                    String imageURL;
+                    JSONObject imageOptions = null;
+                    if (richContent.get("image") instanceof String) {
+                      imageURL = richContent.getString("image");
+                    } else {
+                      imageURL = richContent.getJSONObject("image").getString("url");
+                      imageOptions = richContent.getJSONObject("image").getJSONObject("options");
+                    }
+
+                    boolean thumbnailHidden = false;
+                    if (imageOptions != null && imageOptions.getBoolean("thumbnailHidden")) {
+                      thumbnailHidden = true;
+                    }
+
+                    final Bitmap imageBitmap = loadRemoteImage(imageURL, context);
+                    if (imageBitmap != null) {
+                      NotificationCompat.BigPictureStyle bigPictureStyle = new NotificationCompat.BigPictureStyle()
+                          .bigPicture(imageBitmap)
+                          .setBigContentTitle(message);
+                      if (!hasCustomIcon && !thumbnailHidden) {
+                        // Make the rich content image the thumbnail too if there's no "icon" specified.
+                        // Ref: https://developer.android.com/training/notify-user/expanded#image-style
+                        bigPictureStyle.bigLargeIcon(null);
+                        notificationBuilder.setLargeIcon(imageBitmap);
+                      }
+                      notificationBuilder.setStyle(bigPictureStyle);
+                    }
+                  }
+                }
+              } catch (JSONException e) {
+                // Do nothing. Something is wrong with the user-provided data payload.
```

Let's at least log the exception here so we don't swallow it completely